examples/visual_recognition_v3.py

```diff
@@ -34,21 +33,21 @@
 #
 # cars_positive_examples=image_file), indent=2))
 
-url_result = visual_recognition.classify(images_url=test_url)
+url_result = visual_recognition.classify(parameters=json.dumps({'url': test_url}))
 print(json.dumps(url_result, indent=2))
 
-faces_result = visual_recognition.detect_faces(images_url=test_url)
+faces_result = visual_recognition.detect_faces(parameters=json.dumps({'url': test_url}))
```

Comment:
It's unfortunate that the API for Visual Recognition expects a JSON string. Not sure what can be done to improve that without changing the service interface.

Reply:
:(